101-azure-database-migration-service/azuredeploy.parameters.json

```diff
@@ -1,34 +1,23 @@
-
 {
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
     "contentVersion": "1.0.0.0",
-    "parameters":
-    {
-        "createPublicIP":
-        {
-            "value": false
+    "parameters": {
+        "createPublicIP": {
```

Comment:
remove this and use a defaultValue in the template

Reply:
Done